Tools/AP_Bootloader/bl_protocol.cpp

```diff
@@ -128,6 +128,7 @@
 #define PROTO_DEVICE_FW_SIZE	4	// size of flashable area
 #define PROTO_DEVICE_VEC_AREA	5	// contents of reserved vectors 7-10
 #define PROTO_DEVICE_EXTF_SIZE  6   // size of available external flash
+#define PROTO_DEVICE_BL_GIT_HASH 7  // git hash of bootloader build
```

Comment:
I've also got changes that add reading off the Application Git SHA if `AP_CHECK_FIRMWARE_ENABLED` is enabled in the bootloader (and application)